sce.bat

```diff
@@ -140,7 +140,8 @@ REM set the varible %name% to the resolved repo.
 :print_repo_nicknames
     echo.
     echo each repo has nicknames:
-    echo Core-v4:core-v4, corev4, cv4, c4, c
+set CLARK_OPTIONS="clark" "clrk" "ck" "c"
```

Comment:
remove this line, looks like someone was trying to be cool with multiple cursors in vscode
```suggestion
```

Reply:
what